tileview/src/main/java/com/qozix/tileview/tiles/TileRenderPoolExecutor.java

```diff
@@ -0,0 +1,115 @@
+package com.qozix.tileview.tiles;
+
+import java.lang.ref.WeakReference;
+import java.util.LinkedList;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.Future;
+import java.util.concurrent.LinkedBlockingDeque;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+public class TileRenderPoolExecutor {
+  // Sets the amount of time an idle thread will wait for a task before terminating
+  private static final int KEEP_ALIVE_TIME = 5;
+  private static final TimeUnit KEEP_ALIVE_TIME_UNIT;
+
+  private static final int CORE_POOL_SIZE = Runtime.getRuntime().availableProcessors();
+  private static final int MAXIMUM_POOL_SIZE = Runtime.getRuntime().availableProcessors();
+
+  private final CustomThreadPoolExecutor mExecutor;
+  private final BlockingQueue<Runnable> mQueue;
+  private final BlockingQueue<Future> mFutureList;
+  private WeakReference<TileCanvasViewGroup> mViewGroup;
+
+  private static TileRenderPoolExecutor sInstance = null;
+
+  static {
+    KEEP_ALIVE_TIME_UNIT = TimeUnit.SECONDS;
```

Comment:
this doesn't need to be in a static block, you could just say back on line 14 `private static final TimeUnit KEEP_ALIVE_TIME_UNIT = TimeUnit.SECONDS;`